WF_Proton_Launcher/Main.cpp

```diff
@@ -26,7 +26,7 @@ namespace fs = std::experimental::filesystem;
 #endif
 
 const std::string INDEX_HOST = "origin.warframe.com";
-const std::string CONTENT_HOST = "content.warframe.com";
+const std::string REGISTRY = "";
```

Comment:
I'd remove this entirely, no reason to set up a const definition just for an empty string.

Instead, just do `std::string registry_option;` when declaring the variable.